master/docs/relnotes/0.8.12.rst

```diff
@@ -82,7 +82,7 @@ Features
 
 * GitHub change hook now supports payload validation using shared secret, see :ref:`GitHub-hook` for details.
 
-* Added StashStatusPush status hook for Atlassian Stash
+* StashStatusPush status hook added configurability for both key and name.
```

Comment:
I'm sorry I was not specific: all release notes changes go to `master/docs/relnotes/index.rst`.

And I would appreciate if relnotes change is in a separate commit: this helps porting the change between branches ([docs from trac](http://trac.buildbot.net/wiki/SubmittingPatches)).